test/functional/test_framework/authproxy.py

```diff
@@ -62,6 +63,8 @@ def __init__(self, rpc_error, http_status=None):
 def EncodeDecimal(o):
     if isinstance(o, decimal.Decimal):
         return str(o)
+    if isinstance(o, pathlib.Path):
+        return str(o)
```

Comment:
nit (feel free to ignore for now, probably follow-up material): Now that this function is not handling decimals only anymore, I think it should be renamed from `EncodeDecimal` to something generic?

Reply:
See #28011.